web-admin/src/routes/-/embed/+layout.svelte

```diff
@@ -43,6 +44,9 @@
   $: onProjectPage = !activeResource;
 
   onMount(() => {
+    // Force light mode for embeds
+    themeControl.set.light();
```

Comment:
Does this need to be set in `onMount`? It might avoid a theme flash if this were set prior, outside of `onMount`.